constants/RoutePaths.ts

```diff
@@ -0,0 +1,9 @@
+enum RoutePaths {
+  Home = '/',
+  Explorer = '/explorer',
+  TransactionDetail = '/transaction/:id',
+  BlockDetail = '/blocks/:id',
+  Charts = '/charts',
+}
```

Comment:
Minor but is this `enum` needed? Seems like it could just as easily be an object of string values? (Not a blocker, just trying to understand intent.)

Reply:
[This is the main different between `enum` and `object`](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums)